src/_common/script/Globals.coffee

```diff
@@ -8,3 +8,40 @@ export default class Globals
 		MAIN_CANVAS: 1000
 		CANVAS_RETICLE: 1001
 
+	@STATUS_PLACETILE = 'placeTile'
+
+	@PALETTE = [
+		[0x6D, 0x00, 0x1A, 0xFF]
+		[0xBE, 0x00, 0x39, 0xFF]
```

Comment:
Each grouping of four bytes here represents a single color's RGBA value. Any one color's index within this array -- that color's color code -- is a value that I'll pass around to various parts of the backend architecture, but only the front end will care what the color's RGBA actually is. Since color codes already go from 0 to 31 inclusive, it didn't really make sense for me to map color codes to color RGBAs -- I just use the color code as the index.

Perhaps I could rename this variable to something more clear. `COLOR_RGBA_BY_CODE` perhaps?

Reply:
I wouldn't worry about it.